index.js

```diff
@@ -56,5 +56,5 @@ module.exports = {
       },
     },
   },
-  html: '<label for="{{quote.id}}">{{quote.label}}</label><input type="{{quote.type}}" id="{{quote.id}}" name="{{quote.name}}" value="{{quote.value}}" placeholder="{{quote.placeholder}}" /><label for="{{author.id}}">{{author.label}}</label><input type="{{author.type}}" id="{{author.id}}" name="{{author.name}}" value="{{author.value}}" placeholder="{{author.placeholder}}" /><label for="{{source.id}}">{{source.label}}</label><input type="{{source.type}}" id="{{source.id}}" name="{{source.name}}" value="{{source.value}}" placeholder="{{source.placeholder}}" />',
+  html: '<div class="form--field"><label for="{{quote.id}}">{{quote.label}}</label><input type="{{quote.type}}" id="{{quote.id}}" name="{{quote.name}}" value="{{quote.value}}" placeholder="{{quote.placeholder}}" /></div><div class="form--field"><label for="{{author.id}}">{{author.label}}</label><input type="{{author.type}}" id="{{author.id}}" name="{{author.name}}" value="{{author.value}}" placeholder="{{author.placeholder}}" /></div><div class="form--field"><label for="{{source.id}}">{{source.label}}</label><input type="{{source.type}}" id="{{source.id}}" name="{{source.name}}" value="{{source.value}}" placeholder="{{source.placeholder}}" /></div>',
```

Comment:
Hi @poofichu 

What was this html added for? The system already takes into account when an input plugin has more than one input and it puts different html in there accordingly:

```
<fieldset id="some-quote" class="form--fieldset ">
  <legend class="form--legend">Example quote</legend>
  <p class="form--description">Simple input plugin for quote type</p>

  <label for="03442814-3401-44d3-8898-f80c215c82c0">Some quote</label><input type="text" id="03442814-3401-44d3-8898-f80c215c82c0" name="some-quote--quote" value="" placeholder="Quotation Text"><label for="daf6b27d-a723-4517-b4a2-83047e1d4ace">Some Author</label><input type="text" id="daf6b27d-a723-4517-b4a2-83047e1d4ace" name="some-quote--author" value="" placeholder="Author Name"><label for="695242a1-0962-4ebf-b794-2ba618a9b26b">Some Source Material</label><input type="text" id="695242a1-0962-4ebf-b794-2ba618a9b26b" name="some-quote--source" value="" placeholder="Source Material">

</fieldset>
```

As opposed to a single-input plugin's html:

```
<div id="some-text" class="form--field ">
  <label for="d6444361-cf85-4815-8433-ab341d994249">Some Text</label><input type="text" id="d6444361-cf85-4815-8433-ab341d994249" name="some-text--text" value="" placeholder="Text Goes Here">

  <p class="form--description">I am the Bar Content Type Config text field description</p>
</div>
```

The reason this concerns me is that Punchcard will not have a say in what html might be inside a given plugin. If we want to have styles that work across the board for multi-inputs (or other types of input plugins) we should discuss the need and possibly add this html to the [function in content types which controls the html around input plugins](https://github.com/punchcard-cms/content-types/blob/5550f489f113af109d18b159f3dcb7102b79c830/lib/form/html.js#L127)